support/index.html

```diff
@@ -55,6 +60,13 @@ <h2>Core Contributors</h2>
         <div class="col-xs-10">
             <a href="https://federal.planet.com">Planet Federal</a> (US) works to ensure that the geospatial community reaps the benefits of open source. Formerly the geospatial division of OpenPlans, starting GeoServer in 2001. Established as an independent company in 2013, and acquired by Planet in 2019. Planet Federal employs experts from the open source geospatial community and the broader geospatial field who recognize the power and momentum of open source on the future of the industry. For more information <a href="https://www.planet.com/contact/">contact </a> Planet Federal or visit the website for information about GeoServer and Boundless Suite support.
```

Comment:
I clicked through the expired certificate and it just redirects to https://www.planet.com/markets/federal/ (which looks like it has all the relevant info) so I think you can use that URL instead.